packages/server/graphql/mutations/setTaskEstimate.ts

```diff
@@ -209,17 +212,32 @@ const setTaskEstimate = {
         return {error: {message: 'User no longer has access to Azure DevOps'}}
       }
 
+      let fieldName = SprintPokerDefaults.SERVICE_FIELD_COMMENT.toString()
+      let fieldType = 'string'
       if (!azureDevOpsDimensionFieldMapEntry) {
-        return {error: {message: 'Cannot find the correct field to push changes to.'}}
+        try {
+          const props = {
+            teamId,
+            dimensionName,
+            fieldName: fieldName,
+            fieldId: fieldName,
+            instanceId,
+            fieldType: fieldType,
+            projectKey
+          } as AzureDevOpsFieldMapProps
+          await upsertAzureDevOpsDimensionFieldMap(props)
```

Comment:
-1 We shouldn't update the field here. If the user switches fields, then `updateAzureDevOpsDimensionField` is fired and that should be enough.

Reply:
@Dschoordsch - I had previously updated the database schema to include the workItemType, so that we can capture the AzureDevOps field to update for each work item type.  However, this caused issues in the EstimateStage of the graphql schema at /server/graphql/types/EstimateStage.ts.  At that point, there is no reference to the workItemType which was causing buggy behavior.  Modifying at that level could impact other connections.

Reply:
@Dschoordsch - The behavior that is currently in place when navigating from one work item to another is:

1. If the currently selected field to update belongs to the current (just navigated to) work item, it will set it
2. If the currently selected field does not belong to the current (just navigated to) work item, it will default to comment

Reply:
@Dschoordsch - I wanted to add a little more detail to [this comment](https://github.com/ParabolInc/parabol/pull/6593#discussion_r889179662).  Inside EstimateStage, there is a request for serviceField.  In the resolver, there is branching logic to determine which service/connector is in context.  At this point, we aren't aware of a particular work item type, we only have {teamId, dimensionName, instanceId, and projectKey}.

Reply:
Looking at https://github.com/ParabolInc/parabol/blob/ae1640ed948c80a7ad9fbea887ffa94d579330d1/packages/server/utils/azureDevOps/azureDevOpsFieldTypeToId.ts it appears to me as if there is only 1 canonical voting field per issue type. So regardless of the issue type you're currently voting on, you always have the 3 options:
1) Do Not Update
2) Vote to comment
3) Vote to issue type specific effort field
Thus you would only need to store an enum with 3 values and then just on voting/displaying you need to translate what 3) actually means.

Reply:
@Dschoordsch - you're right.  Originally, we were going to make sure we supported custom templates, where admins could create many fields.  So we were going to take the base fields and then expand custom fields from a list.  Right now, that list is static, but to support custom templates/multiple fields, all you would need to do is replace that list implementation with something dynamic like json stored in the database.  

However, the number of customers that I have worked with that have implemented custom templates over the years have been very few.  If you would like to replace the implementation with a singular effort field and map, I can do that.

Reply:
We should do one of those, either change to a singular field in the effort field map (UI should still show correct name based on issue type) or we store the setting per issue type.
I think the first option would be much less work, so I think we should do that now.

Reply:
I made the change to support a setting per issue type